app/controllers/application_controller.rb

```diff
@@ -2,7 +2,7 @@ class ApplicationController < ActionController::API
   include Pundit::Authorization
   after_action :verify_authorized
 
-  # temporary current_user testing stub until we add in authentication
+  
   def current_user
     @current_user ||= self.authenticate_user
   end
```

Comment:
When you say, "When we add authentication" do you mean the authentication using JWT?

Reply:
Yup, and that pseudo is deleted.  I was mocking and stubbing prior to introduction of JWT, and finally got rid of the remnants.